CMakeLists.txt

```diff
@@ -195,12 +195,12 @@ list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
 #-----------------------------------------------------------------------------
 # Srep
-set(extension_name "SlicerSkeletalRepresentation")
-set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/SRep")
+set(extension_name "SRep")
```

Comment:
Almost, the name of the extension is still `SlicerSkeletalRepresentation` but the source folder is `SRep`. I suggest to simply keep the file how it was.

Reply:
You could add a comment like 

```
set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/SRep") # Use shorter extension name to issues with long paths on windows
```

Reply:
Before this patch we have long names plus inconsistency.
![image](https://user-images.githubusercontent.com/3021667/55247194-52e85d80-521d-11e9-8a9b-919a2cef1e01.png)
![image](https://user-images.githubusercontent.com/3021667/55247149-3ba97000-521d-11e9-8516-4b93b8e9d6de.png)

 I suggest we take one of the both approaches on this patch, with your comment.

Reply:
I added a short_extension_name, and use it for the FetchContent function and the `_SOURCE_DIR`. Let me know what you think. Also added the comment explaining why

Reply:
Well done :+1: